tools/schemacode/schemacode/validator.py

```diff
@@ -396,32 +402,26 @@ def validate_all(
     tracking_schema = deepcopy(regex_schema)
     paths_list = _get_paths(bids_paths, pseudofile_suffixes=pseudofile_suffixes)
     tracking_paths = deepcopy(paths_list)
-    if debug:
-        itemwise_results = []
+    itemwise_results = []
     matched = False
     match_listing = []
     for target_path in paths_list:
-        if debug:
-            print(f"Checking file `{target_path}`.")
-            print("Trying file types:")
+        lgr.debug(f"Checking file `{target_path}`.")
```

Comment:
here and elsewhere use %-interpolation instead of f-strings to avoid possible errors later on if e.g. `target_path` does contain some `%` and logger would start %-interpolating, thus

```suggestion
        lgr.debug("Checking file `%s`.", target_path)
```